xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -345,9 +410,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Writes all resources to the output stream in the system default format.</summary>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+If an exception occurs during object serialization or during IO, the .resources file is closed and deleted, since it is most likely invalid.
```

Comment:
My suggested change based on @ericstj comment for the summary:

```suggestion
## Remarks

If the resources added to the writer can be represented in the system default format understood by the <xref:System.Resources.ResourceReader>, they will be written as such. If the resources require runtime serialization other than that supported by the system default format (<xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter>) then we will write them using the format understood by <xref:System.Resources.Extensions.DeserializingResourceReader>.

If an exception occurs during object serialization or during IO, the .resources file is closed and deleted, since it is most likely invalid.
```

Reply:
Should be runtime deserialization, otherwise seems fine